src/utils/constants/transaction.ts

```diff
@@ -0,0 +1,11 @@
+
+import { newMonetaryAmount } from '@interlay/interbtc-api';
+
+import { GOVERNANCE_TOKEN } from 'config/relay-chains';
+
+// FIXME: account for transaction fees not with a hardcoded value
+const TRANSACTION_FEE_AMOUNT = newMonetaryAmount(0.01, GOVERNANCE_TOKEN);
```

Comment:
I created this as a constant for now for a maintaining purpose.
I'd suggest using this constant whenever we need to consider the transaction fee.
If later on we manage to calculate the transaction fee more correctly without hardcoding like so, then we will be able to get aware of the change cases or just update this file to reflect that change.

@nud3l 
For now, this is the only use case but I can expect a few more cases e.g. transfer. I have not checked how the validation in the transfer form is configured but I think we should double-check it there. @tomjeatt
cc @alexeiZamyatin 
Especially, looking at other DeFi UX examples, they have max buttons to help users input their maximum amounts via one-click button. If we also want to have that UX feature later on, this constant also will be used.

Reply:
Please don't put this in the constants but leave it where it was. The transaction fees for different operations will be quite different and this value should only be used on staking. We can handle it in a later PR and in fact, #179 is the first step to get transaction fees included in the app.

Reply:
@nud3l 
OK, let me update it properly.